data/json/monster_factions.json

```diff
@@ -123,5 +123,20 @@
         "type" : "MONSTER_FACTION",
         "name" : "spider_cellar",
         "base_faction" : "spider"
+    },{
+        "type" : "MONSTER_FACTION",
+        "name" : "mutant",
+        "neutral" : ["vermin"],
```

Comment:
Adding vermin to neutral is redundant - the default faction is already neutral to them.